### ISSUE TITLE
Steps should appear as ordered lists

### ISSUE THREAD

**Issue by desigonz** [cityofaustin/techstack#3186]
Steps are ordered lists, and they should appear that way to a screenreader—but currently they aren't doing that.

**Acceptance criteria**
- [x] A step appears as an `<ol>` when you inspect element

**Comment by desigonz** [comment]
@briaguya Am I missing anything on this acceptance criteria?

**Comment by chiaberry** [comment]
@desigonz PR has been approved, just needs QA https://janis-3186-steps-are-ol.netlify.com/en/health-safety/health-records-certificates-2/birth-death-certificates/birth-and-death-certificates-test

**Comment by desigonz** [comment]
@chiaberry Looks great, QA complete!
